docs/pages/examples/dashboard/cards.tsx

```diff
@@ -107,7 +101,7 @@ const Item: FC<ItemProps> = memo(
 
           <Spacer />
 
-          <Icon as={icon} color="muted" {...iconProps} />
+          <Icon color="muted" size="2xl" {...iconProps} />
```

Comment:
The size seems to have changed

before
<img width="1318" alt="image" src="https://github.com/yamada-ui/yamada-ui/assets/77735745/3fa7a92b-9f93-4713-8814-186d8187c47b">


after
<img width="1318" alt="image" src="https://github.com/yamada-ui/yamada-ui/assets/77735745/eccf7331-0c6a-4fea-8237-e64d6e0b2fa8">

Reply:
@fffk10 

That change is intentional. The icon was too small.